### PR TITLE
Update kubernetes version to 1.13.1

### PR DESCRIPTION
**Pull request by Madhu-1**
* CSI is updated to v1.0.0
* Removed CRD for driver register and node info

Signed-off-by: Madhu Rajanna <mrajanna@redhat.com>

### CODE REVIEW

**Comment by Madhu-1** [comment]
currently, kubespray supports kubernetes 1.12.x releases

once we get kube 1.13.0 support in kubespray we can merge this PR.

**Comment by Madhu-1** [comment]
```
[mrajanna@localhost deploy]$ kubegcs get po
NAME                                   READY   STATUS             RESTARTS   AGE
anthill-58b9b9b6f-zfj85                0/1     CrashLoopBackOff   6          12m
csi-attacher-glusterfsplugin-0         2/2     Running            0          4m20s
csi-nodeplugin-glusterfsplugin-572j6   2/2     Running            1          4m20s
csi-nodeplugin-glusterfsplugin-d4fqh   2/2     Running            0          4m20s
csi-nodeplugin-glusterfsplugin-pqm22   2/2     Running            1          4m20s
csi-provisioner-glusterfsplugin-0      3/3     Running            0          4m20s
etcd-2k82znllv9                        1/1     Running            0          9m52s
etcd-6v8lvfdnbh                        1/1     Running            0          12m
etcd-9cxvgm79wv                        1/1     Running            0          10m
etcd-operator-77bfcd6595-6mssd         1/1     Running            0          12m
gluster-kube1-0                        1/1     Running            0          9m42s
gluster-kube2-0                        1/1     Running            1          9m41s
gluster-kube3-0                        1/1     Running            1          9m40s
[mrajanna@localhost deploy]$ kube get pvc
NAME                 STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS    AGE
gcs-example-volume   Bound    pvc-3a7db00e-10ab-11e9-b504-52540001049b   1Gi        RWX            glusterfs-csi   81s
[mrajanna@localhost deploy]$ kube get po
NAME          READY   STATUS    RESTARTS   AGE
gcs-example   1/1     Running   0          87s
```

```
[mrajanna@localhost deploy]$ kube version
Client Version: version.Info{Major:"1", Minor:"13", GitVersion:"v1.13.1", GitCommit:"eec55b9ba98609a46fee712359c7b5b365bdd920", GitTreeState:"clean", BuildDate:"2018-12-13T10:39:04Z", GoVersion:"go1.11.2", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"13", GitVersion:"v1.13.1", GitCommit:"eec55b9ba98609a46fee712359c7b5b365bdd920", GitTreeState:"clean", BuildDate:"2018-12-13T10:31:33Z", GoVersion:"go1.11.2", Compiler:"gc", Platform:"linux/amd64"}
```

**Comment by Madhu-1** [comment]
@JohnStrunk PTAL. dont merge this PR till we support csi 1.0 in gluster-csi-driver

**Comment by Madhu-1** [comment]
once we get the container out of https://github.com/gluster/gluster-csi-driver/pull/133 we can merge this one.

**Comment by humblec** [comment]
FYI , The CSI patch is merged.

**Comment by Madhu-1** [comment]
@kshlm @JohnStrunk PTAL

File: deploy/prepare.sh [hunk 1]
```diff
@@ -37,6 +37,10 @@ fi
 echo "Ensuring kubespray is present"
 git submodule --quiet init
 git submodule --quiet update
+cd kubespray
+#check out to support kube 1.13.1
+git checkout 39d750
```
Comment (kshlm) [inline deploy/prepare.sh]: This shouldn't be required. The `git submodule (init|update)` commands above should checkout the correct git hash  as you've also submodule git hash.

**Comment by Madhu-1** [comment]
@atinmu @kshlm @aravindavk can we review this PR on priority, current GCS deployment is failing as CSI docker image is updated to CSI 1.0.0

**Comment by kshlm** [comment]
Thanks @Madhu-1 . I'm currently doing a deploy with this. Will merge once it finishes successfully.

File: deploy/prepare.sh (hunk 1 above)
Comment (JohnStrunk) [reply deploy/prepare.sh]: @kshlm @Madhu-1 I this is overriding the submodule commit... I think this needs to be removed.
The submodule commands may also not be what we want... not really sure. We need to make sure that a commit of gcs w/ a particular kubespray hash is sufficient to advance the version w/o updating the hash in the script.

Comment (Madhu-1) [reply deploy/prepare.sh]: @JohnStrunk https://github.com/gluster/gcs/pull/105 PTAL

**Pull request merged**